front/app/modules/commercial/insights/hooks/useInsightsInputs.ts

```diff
@@ -42,6 +42,7 @@ const useInsightsInputs = (
         'page[number]': queryParameters?.pageNumber || 1,
         'page[size]': queryParameters?.pageSize || defaultPageSize,
       },
+      skipSanitizationFor: ['category'],
```

Comment:
I wonder if we shouldn't add this to the service directly, rather than the hook?
This is where I edited the caching behaviour for example, so it would make sense to keep it consistent - one way or the other. Is there a preferred way of where to keep this options atm?

Reply:
Makes sense, moved it ! :+1:  I wouldn't say there is a preferred way, they're not used that often so to me it's on a case by case basis, and under change because moving from resources components to hooks